libs/README.md

```diff
@@ -17,8 +17,30 @@ downward line means "is required by"):
 We recommend building with conan 2, and only conan 2 builds are used on the CI.
 However, the libraries may be built using either conan 1 or conan 2.
 
+The build requires `cmake` version 3.26 or above.
+
+
 ## conan 1
 
+```shell
+pip install conan~=1.59
+```
+
+Create a new profile called `tket`:
+
+```
+conan profile detect --name tket
```

Comment:
```suggestion
conan profile new tket --detect

Reply:
> There is a difference for conan 1 and 2

Yes, because there is no need to create a custom named profile as we're not updating it: in conan 2 the Linux `libcxx` configuration is correctly set with `detect`. It is easiest to use the `default` profile.

Reply:
> I think this is in the conan 1 part and `conan profile detect` is not working for that? (Or I am doing something wrong locally in conan 1?)

Sorry yes I'll fix that.

Reply:
> Are we sure that we don't want to use a specific profile to allow for some individual setting like debug mode or something like that in conan 2?

Individuals can do that if they want of course -- but these instructions aren't meant to cover all cases, just enough to do a build.

Reply:
> > I think this is in the conan 1 part and `conan profile detect` is not working for that? (Or I am doing something wrong locally in conan 1?)
> 
> Sorry yes I'll fix that.

Done.